src/block-management/block-holder.js

```diff
@@ -53,6 +69,28 @@ export default class BlockHolder extends React.Component<PropsType, StateType> {
 					style={ style }
 				/>
 			);
+		} else if ( this.props.name === 'aztec' ) {
+			return (
+				<RCTAztecView
+					accessibilityLabel="aztec-view"
+					style={ [
+						styles[ 'aztec-editor' ],
+						{ minHeight: Math.max( _minHeight, this.state.aztecheight ) },
+					] }
+					text={ this.props.attributes.content }
+					onContentSizeChange={ ( event ) => {
+						this.setState( { ...this.state, aztecheight: event.nativeEvent.contentSize.height } );
+					} }
+					onChange={ ( event ) => {
+						this.props.onChange( this.props.uid, {
+							...this.props.attributes,
+							content: event.nativeEvent.text,
+						} );
+					} }
```

Comment:
I've been considering this a while and opening here for discussion - what do you think about using the `ReactAztecFocusEvent` and `ReactAztecEndEditingEvent` events to control when to update the state only, instead of making each TextChange event reflect on the RN kept state in this case?. I understand this is counterintuitive in terms of the React logic, but it rings as something plausible given the performance issues, and is a "pure JS" solution (as opposed to involving Java and more Js/Java interaction controlling code).
I've made tests here just changing `onChange`  for `onEndEditing` in [line 84 up here](https://github.com/wordpress-mobile/gutenberg-mobile/pull/60/files#diff-2004b9687d71de0f0a9816269bb4a6a2R84) and it seems to work pretty well (no more lagging / weird refreshing issues).  This, without the code in #62 and https://github.com/wordpress-mobile/react-native-aztec/pull/24  cc @daniloercoli @hypest

Reply:
👋 @mzorz . I'm not well informed yet on the lifecycle events in RN, and in light of https://github.com/wordpress-mobile/react-native-aztec/pull/24 that fixes the editing consistency issue atm, I'd say let's leave the proposal for `onEndEditing` for a separate PR. I'm actually sure we will visit that since yeah, it is probably suboptimal to update the Redux state on every key press.